apps/ewallet/lib/ewallet/gates/blockchain_transaction_gate.ex

```diff
@@ -150,10 +150,8 @@ defmodule EWallet.BlockchainTransactionGate do
   end
 
   defp set_blockchain(attrs) do
-    adapter = BlockchainHelper.adapter()
-
     attrs
-    |> Map.put("blockchain_identifier", adapter.helper.identifier)
+    |> Map.put("blockchain_identifier", BlockchainHelper.identifier())
```

Comment:
In what cases do we go through adapter.helper and when to call directly?

Reply:
This is some code from Tibo PR where he didn't have the `BlockchainHelper` yet.